Zend/tests/partial_application/variation_nocall_002.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Closure application variation no call order of destruction
```

Comment:
“order of destruction”. Should we have a class with a destructor that prints something here?